scripts/Calibration/Examples/TubeCalibDemoMaps_All.py

```diff
@@ -500,8 +502,8 @@ def completeCalibration(filename):
     mantid.ApplyCalibration(Workspace=CalibInstWS, PositionTable=calibrationTable)
 
     # == Save workspace ==
-    # SaveNexusProcessed( CalibInstWS, path+'TubeCalibDemoMapsResult.nxs',"Result of Running TCDemoMaps.py")
-    # print "saved calibrated workspace (CalibInstWS) into Nexus file TubeCalibDemoMapsResult.nxs"
+    # mantid.SaveNexusProcessed(CalibInstWS, path+'TubeCalibDemoMapsResult.nxs', "Result of Running TCDemoMaps.py")
```

Comment:
The print statements, both uncommented and commented, have been changed to use the Python 3 syntax unless I missed some.